lib/chiketto/attr_attribute.rb

```diff
@@ -20,7 +20,7 @@ def attr_date(*args)
             return date if date.nil?
 
             if date.is_a?(Hash)
-              date = date['local']
+              date = date['utc']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.